wpilibNewCommands/src/main/java/edu/wpi/first/wpilibj2/command/PIDSubsystem.java

```diff
@@ -25,12 +25,23 @@
    * Creates a new PIDSubsystem.
    *
    * @param controller the PIDController to use
+   * @param initialPosition the initial setpoint of the subsystem
    */
-  public PIDSubsystem(PIDController controller) {
+  public PIDSubsystem(PIDController controller, double initialPosition) {
     requireNonNullParam(controller, "controller", "PIDSubsystem");
+    setSetpoint(initialPosition);
     m_controller = controller;
```

Comment:
Should this use the same m_controller = requireNonNullParam idiom as the other change?

Reply:
Yes, it should, I'll change that.